<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimpleReplace</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionFileSelectDirectory"/>
    <addaction name="separator"/>
    <addaction name="actionFileFindOnly"/>
    <addaction name="actionFileFindAndReplace"/>
    <addaction name="separator"/>
    <addaction name="actionFileQuit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionSettingsPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelpDocumentation"/>
    <addaction name="actionHelpAbout"/>
    <addaction name="actionHelpAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuProfile">
    <property name="title">
     <string>Profile</string>
    </property>
    <addaction name="actionProfileNew"/>
    <addaction name="actionProfileSave"/>
    <addaction name="actionProfileSaveAs"/>
    <addaction name="actionProfileRename"/>
    <addaction name="actionProfileDuplicate"/>
    <addaction name="actionProfileDelete"/>
    <addaction name="separator"/>
    <addaction name="actionProfileImport"/>
    <addaction name="actionProfileExport"/>
    <addaction name="separator"/>
    <addaction name="actionProfileManageProfiles"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuProfile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionHelpDocumentation">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::HelpFaq"/>
   </property>
   <property name="text">
    <string>Documentation</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::AboutRole</enum>
   </property>
  </action>
  <action name="actionHelpAbout">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::HelpAbout"/>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionSettingsPreferences">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentProperties"/>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+,</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::PreferencesRole</enum>
   </property>
  </action>
  <action name="actionFileSelectDirectory">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::FolderOpen"/>
   </property>
   <property name="text">
    <string>Select Directory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionFileQuit">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ApplicationExit"/>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::QuitRole</enum>
   </property>
  </action>
  <action name="actionProfileNew">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ListAdd"/>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionProfileSave">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentSave"/>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionProfileSaveAs">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentSaveAs"/>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionProfileRename">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MailMessageNew"/>
   </property>
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
  <action name="actionProfileDelete">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditDelete"/>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionProfileImport">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionProfileExport">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentSend"/>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionProfileManageProfiles">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentProperties"/>
   </property>
   <property name="text">
    <string>Manage Profiles</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionFileFindOnly">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditFind"/>
   </property>
   <property name="text">
    <string>Find Only</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionFileFindAndReplace">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaPlaybackStart"/>
   </property>
   <property name="text">
    <string>Find and Replace</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionProfileDuplicate">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditCopy"/>
   </property>
   <property name="text">
    <string>Duplicate</string>
   </property>
  </action>
  <action name="actionHelpAboutQt">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/Qt_Icon_32x32.png</normaloff>:/resources/Qt_Icon_32x32.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::AboutQtRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
